src/modules/CRUD/services/CRUDProvider.js

```diff
@@ -47,41 +50,73 @@ export class CRUDProvider extends React.Component {
     } catch (e) {
       this.setState(state => ({
         ...state,
-        errors: { ...state.errors, code: e.message },
+        errors: { ...state.errors, message: e.message },
       }));
     }
   }
 
   getFeaturesList = async layerId => {
+    if (!layerId) {
+      return;
+    }
     try {
-      const { results: featuresList = [] } = await fetchFeaturesList(layerId);
-      this.setState({ featuresList });
+      const { results: featuresList } = await fetchFeaturesList(layerId);
+      this.setState(state => ({
```

Comment:
As seen together, the returned object could be extracted in a small pure function, and maybe reused in cache block.

*And the best would be to have a way to call `setState` after the catch block (to avoid unexpected silent errors)*